inference-engine/src/legacy_api/src/graph_transformer.cpp

```diff
@@ -219,7 +219,9 @@ static std::vector<std::string> skipConstInfer = {
     "Convolution", // Const inference function for Convolution is not implemented
     "Eltwise",  // Const inference function for Eltwise is not implemented
     "FullyConnected",
-    "Squeeze"
+    "Squeeze",
```

Comment:
Suitable for a temporary solution
@dmitry-gorokhov @ilya-lavrenov why do we need graph_transformer? Does he now perform any functions that are not implemented on nGraph side?

Reply:
We still have a lot of tests which are based on IR v7. We need to update the tests to use ngraph representation and remove transformer usage later (as it's done for GPU)